phpBB/docs/events.md

```diff
@@ -854,27 +860,6 @@ posting_editor_bbcode_status_after
 * Since: 3.1.4-RC1
 * Purpose: Add content after bbcode status
 
-posting_editor_buttons_after
```

Comment:
can we avoid deleting events? If not we need to create a list some where of events we removed.